Sources/AudioKitUI/Visualizations/FFTView.swift

```diff
@@ -1,16 +1,31 @@
 // Copyright AudioKit. All Rights Reserved. Revision History at http://github.com/AudioKit/AudioKitUI/
 
+import Accelerate
 import AudioKit
 import SwiftUI
 
+enum FFTXScale {
+    case linear
+    case log
+}
+
+enum FFTBinAggregation {
+    case max
+    case average
+}
+
 class FFTModel: ObservableObject {
-    @Published var amplitudes: [Double?] = Array(repeating: nil, count: 50)
+    @Published var amplitudes: [Float?] = Array(repeating: nil, count: 50)
     var nodeTap: FFTTap!
     private var FFT_SIZE = 2048
     var node: Node?
     var numberOfBars: Int = 50
```

Comment:
I know this is not even one of your changes, but maybe rename this `barCount`. We try to use plural variables for arrays. I hate the `numberOf` prefix.